linux/scripts/reconf.sh

```diff
@@ -53,8 +53,10 @@ for proj in ${autotool_projects}; do
 done
 
 if [ "${extra_project}" == "keyman-config" ]; then
+    majorvers=`cat ../resources/VERSION.md | cut -d . -f 1`
     cd keyman-config/keyman_config
     sed "s/_VERSION_/${newvers}/g" version.py.in > version.py
+    sed "s/_MAJORVERSION_/${majorvers}/g" downloadkeyboard.py.in > downloadkeyboard.py
```

Comment:
I would prefer to see the major version created as an additional constant in version.py.in and just referenced from downloadkeyboard.py, rather than preprocessing downloadkeyboard.py.

Reply:
it now gets the full majorversion from resources/VERSION.md to use in the keyman.com/go link